Scripts/AccessDataWaitForJobResult/AccessDataWaitForJobResult.py

```diff
@@ -0,0 +1,55 @@
+import demistomock as demisto
+from CommonServerPython import *
+from time import sleep
+
+timeout = 960
+interval = 10
+
+# Constant and mandatory arguments
+
+caseid = demisto.get(demisto.args(), 'caseid')
+jobid = demisto.get(demisto.args(), 'jobid')
+
+feDone = False
+
+# Poll stage
+status = None
+sec = 0
+ec = {}  # type: dict
+resp = None
+sleep(10)  # small waitto be sure job is submitted, otherwise we get 404
```

Comment:
this is not the way we usually do polling/long commands.
the reason for that it holds docker on sleep . Instead we create playbook and use GenericPlaybook inside.
Example Joe Security:
![Detonate_File_-_JoeSecurity_Sat_Oct_05_2019](https://user-images.githubusercontent.com/7270217/66251606-a0a8ca00-e75a-11e9-95f1-c2d6ceaa6e6b.png)
We can have a session on how to do it